_docHome.md

```diff
@@ -0,0 +1,150 @@
+
+<DocHomeCardContainer>
+
+<DocHomeCard href="https://docs.pingcap.com/tidb/stable" icon="doc1" label="TiDB Docs">
+
+See the documentation of TiDB
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/tidb/stable/quick-start-with-tidb" icon="doc5" label="Get Started with TiDB">
+
+Walks you through the quickest way to get started with TiDB
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup" icon="cloud7" label="Deploy a Local TiDB Cluster">
+
+Learn how to deploy TiDB locally in production
+
+</DocHomeCard>
+
+</DocHomeCardContainer>
+
+The open source TiDB platform released under the Apache 2.0 license, and supported by the community
+
+<a href="https://en.pingcap.com/download/" class="button" target="_blank" referrerpolicy="no-referrer-when-downgrade">Download</a>
+
+</DocHomeSection>
+
+<DocHomeSection label="Developers" anchor="developers" id="developers">
+
+<DocHomeCardContainer>
+
+<DocHomeCard href="https://docs.pingcap.com/tidb/stable/dev-guide-overview" icon="doc8" label="Developer Guide">
+
+Docs for application developers
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/tidb/stable/dev-guide-sample-application-java" icon="java" label="Java">
+
+Java applications can connect to and communicate with TiDB through JDBC or Swing, or with ORM frameworks like Hibernate.
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/tidb/stable/dev-guide-outdated-for-go-sql-driver-mysql" icon="go" label="Golang">
+
+Golang applications can connect to and communicate with TiDB through a native driver and ORM frameworks.
+
+</DocHomeCard>
+
+</DocHomeCardContainer>
+
+</DocHomeSection>
+
+<DocHomeSection label="More resources" anchor="resources" id="resources">
+
+<DocHomeCardContainer>
+
+<DocHomeCard href="https://en.pingcap.com/education/" icon="cloud1" label="PingCAP Education">
+
+Well-designed online courses and instructor led-training to teach you TiDB and TiDB Cloud
+
+</DocHomeCard>
+
+<DocHomeCard href="https://en.pingcap.com/community/" icon="doc9" label="Community">
+
+Join us on Slack or become a contributor
+
+</DocHomeCard>
+
+<DocHomeCard href="https://en.pingcap.com/blog/" icon="doc10" label="Blog Posts">
+
+Learn great articles about TiDB and TiDB Cloud
+
+</DocHomeCard>
+
+<DocHomeCard href="https://en.pingcap.com/videos/" icon="doc11" label="Videos">
+
+See a compilation of short videos describing TiDB and a variety of use cases
+
+</DocHomeCard>
+
+<DocHomeCard href="https://en.pingcap.com/event/" icon="events" label="Events">
+
+Learn events about PingCAP and the community
+
+</DocHomeCard>
+
+<DocHomeCard href="https://en.pingcap.com/ebook-whitepaper/" icon="papers" label="eBooks & Papers">
+
+Download eBooks and Papers
+
+</DocHomeCard>
+
+<DocHomeCard href="https://ossinsight.io/" icon="cloud6" label="OSS Insight">
+
+A powerful insight tool that can help you analyze in depth any single GitHub repository
```

Comment:
This tool also can compare repos, so I suggest removing "single" to avoid misunderstanding.
```suggestion
A powerful insight tool that can help you analyze in depth any GitHub repository
```